staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -275,6 +275,10 @@ type CPU struct {
 	// the emulator thread on it.
 	// +optional
 	IsolateEmulatorThread bool `json:"isolateEmulatorThread,omitempty"`
+	// OvercommitVCPUs when set to false, requests the scheduler to allocate enough CPUs to the compute container
+	// so there will be no overcommitment of virtual CPUs. (true by default)
+	// +optional
+	OvercommitVCPUs *bool `json:"overcommitVCPUs,omitempty"`
```

Comment:
Are we sure this is necessary?  I'd imagine that explicitly setting the cpu requests/limits on the VMI spec would be the thing that overrides the global overcommit option.

Reply:
It's not necessary, but calculating the correct number of vcpus can be "tricky" (not really, but you need to multiply sockets * cores * threads)
Second, no one just does that. This is the main reason for this PR. Users normally set vCPUs and don't set CPU requests...

Reply:
I'll get rid of this.